.github/workflows/release.yml

```
@@ -2,96 +2,38 @@ name: Release
 on:
   push:
     tags:
-      - '[2-9].[0-9]+.[0-9]+'
-      - '[2-9].[0-9]+.[0-9]+-*'
+      - '[0-9].[0-9]+.[0-9]+'
+      - '[0-9].[0-9]+.[0-9]+-*'
```

Major needs a `+` for when it goes to 2 digits.
```suggestion
      - '[0-9]+.[0-9]+.[0-9]+'
      - '[0-9]+.[0-9]+.[0-9]+-*'
```